### PR TITLE
[Collect] Setup A/B test

### PR DESCRIPTION
**Pull request by l2succes**
This PR sets up the AB test between the new and old collect page.
Related: https://artsyproduct.atlassian.net/browse/GROW-885

(Because of the recent introduction of prettier there's a lot of formatting changes in this PR as well)

### CODE REVIEW

**Comment by eessex** [comment]
You'll need to add a line at the top level where the new app renders to show that the test was viewed -- 
`splitTest('new_collect_page').view()`

Also I really think we should have some tests in place before launching a new app to the public, even if it is a limited segment. I see none covering Reaction components or Force.

**Comment by l2succes** [comment]
Yup, good catch, I'll add that line. And yes, let's get some tests coverage on this.

**Comment by l2succes** [comment]
@eessex Let's hold off on merging this until we get the rest in as it'll launch the test

**Comment by l2succes** [comment]
@damassi Once this PR is merged, the new collect page will be mounted on `/collect` but only for 50% of users. We'll monitor metrics until BNMO launch and roll it out to 100%

**Comment by eessex** [comment]
For test error try adding something like here - https://github.com/artsy/force/blob/33ece666e9df1cf32f9c6cbe7914811815a860cc/src/desktop/apps/editorial_features/components/gucci/components/__tests__/App.test.js#L9

File: src/desktop/apps/collect2/maybeShowOldCollect.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const maybeShowOldCollect = (_req, res, next) => {
+  const { NEW_COLLECT_PAGE } = res.locals.sd
+
+  if (NEW_COLLECT_PAGE === "old") {
```
Comment (eessex) [inline src/desktop/apps/collect2/maybeShowOldCollect.js]: I think you mean to use the new env var here, `NEW_COLLECT_PAGE_ENABLED`

File: src/desktop/components/split_test/running_tests.coffee [hunk 2]
```diff
@@ -45,4 +45,13 @@ module.exports = {
     weighting: 'equal'
     edge: 0
     control_group: 9
+  new_collect_page:
+    key: 'new_collect_page',
```
Comment (eessex) [inline src/desktop/components/split_test/running_tests.coffee]: Maybe this wants to be commented out as well until we start the test, these cookies will be set even if they are not used.

Comment (l2succes) [reply src/desktop/components/split_test/running_tests.coffee]: Right, it's ok that the group bucket cookie gets set. It won't skew our test results as the impression tracking is disabled, which is what matters.

File: src/desktop/apps/collect2/maybeShowOldCollect.js (hunk 1 above)
Comment (eessex) [reply src/desktop/apps/collect2/maybeShowOldCollect.js]: ah ok misread below.

**Comment by damassi** [comment]
Sorry about the Prettier stuff, there's a PR that is waiting to be merged here: https://github.com/artsy/force/pull/2919

**Comment by l2succes** [comment]
@damassi Thanks for pulling the trigger on setting up Prettier in force 🙏🏾

**Pull request merged**